vaxrank/gene_pathway_check.py

```diff
@@ -0,0 +1,125 @@
+# Copyright (c) 2018. Mount Sinai School of Medicine
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import, print_function, division
+from collections import OrderedDict
+
+import pandas as pd
+
+
+_ENSEMBL_GENE_ID = 'Ensembl Gene ID'
+_MUTATION = 'Mutation'
+
+_IFG_RESPONSE = 'interferon_gamma_response'
+_CLASS_I_MHC = 'class1_mhc_presentation_pathway'
+_DRIVER_GENE = 'cancer_driver_gene'
+_DRIVER_VARIANT = 'cancer_driver_variant'
+
+
+class GenePathwayCheck:
```

Comment:
Something about calling this a "pathway check" seems very odd to me.

Reply:
Until we stop supporting Python 2, inherit from `object`: 

https://forum.dabeaz.com/t/inheriting-from-object/161